### ISSUE TITLE
content is not removable if the last block is code-block

### ISSUE THREAD

**Issue by andrewfan** [withspectrum/draft-js-markdown-plugin#80]
Can not remove content after selecting everything (cmd +a) when the last block is `code-block`

https://www.dropbox.com/s/mdvn2pqa52yqufl/Screenshot%202018-05-21%2015.35.40.png?dl=0

**Comment by mxstbr** [comment]
This is a duplicate of #66 I think

**Comment by andrewfan** [comment]
@mxstbr I don't think that it is the duplicate of https://github.com/withspectrum/draft-js-markdown-plugin/issues/66. (I left comment there and I think that it should not be an issue any more)

Hiting `backspace` does not trigger `click` event, so `onClickOutside ` https://github.com/withspectrum/draft-js-markdown-plugin/blob/master/src/components/Code/index.js#L78 is not even called

**Comment by mxstbr** [comment]
Oh sorry, that's true, good point. Reopening!
